packages-content-model/roosterjs-content-model-core/lib/coreApi/createEditorContext.ts

```diff
@@ -30,3 +39,15 @@ function checkRootRtl(element: HTMLElement, context: EditorContext) {
         context.isRootRtl = true;
     }
 }
+
+function getRootComputedStyle(core: StandaloneEditorCore) {
+    const document = core.contentDiv.ownerDocument;
+    let rootElement: HTMLElement | undefined;
+    document.childNodes.forEach(node => {
```

Comment:
can we just use document.body here?

Reply:
I think the root most of the time will be the <html> tag. Maybe document.body.parentElement?